frontend/src/components/eMIB/OrganizationalStructure.jsx

```diff
@@ -47,95 +50,190 @@ class OrganizationalStructure extends Component {
     this.setState({ showPopupBox: false });
   };
 
-  // loads the markdown content (english and french versions)
-  componentWillMount = () => {
+  // populates the tree view array with people names
+  populateTreeArray = response => {
+    const responsePrefixEn =
+      response.background.en.background[0].tree_view[0].organizational_structure_tree_child;
+    const responsePrefixFr =
+      response.background.fr.background[0].tree_view[0].organizational_structure_tree_child;
+    // populating names in English
+    for (let i = 0; i < responsePrefixEn.length; i++) {
```

Comment:
Comment1 feel like this and the next loop could be in their own functions to conserve space (esp since you will probably want to populate group and level from within this loop (See Comment2)

Reply:
talked offline, we will go ahead with the hybrid load from DB/hard coding for now and shift to building treeView entirely from the DB in a future PR